sampo/structurator/__init__.py

```diff
@@ -1,4 +1,3 @@
 from sampo.structurator.base import graph_restructuring, STAGE_SEP
 from sampo.structurator.graph_insertion import graph_in_graph_insertion
 from sampo.structurator.light_modifications import work_graph_ids_simplification
-from sampo.structurator.base import graph_restructuring
```

Comment:
Why?

Reply:
because this is a repeat of the import that was already done in the first line